File: Tests.Data/Output/HTML/Syntax Highlighting/C#/Attributes - Input.cs

/*
	Topic: Attributes

		(code)

		[assembly: GlobalAttribute]
		[module: GlobalAttribute2("x")]
		[UntargetedAttribute]  // Comment
		[UntargetedAttribute2("x", false)]
		class TestClass
			{

			[Flags]
			enum x { a, b, c };

			}

		(end)


	Topic: Attributes vs. Arrays

		(code)

		[Test, Category("x")]
		void Function ([param: something][something else] string[][] x)
			{
			x = new string[12];
			x[variable] = "";
			}

		(end)

*/

File: Tests.Data/Output/HTML/Syntax Highlighting/C/Attributes - Input.cpp

/* Topic: Simple Attributes
	_____________________________________________

	--- Code
	[[AttributeA]]
	[[AttributeB()]]
	[[AttributeC(12, "string")]]
	---
*/


/* Topic: Attributes with Namespaces
	_____________________________________________

	--- Code
	[[Namespace::AttributeA]]
	[[Namespace::AttributeB()]]
	[[Namespace::AttributeC(12, "string")]]
	---
*/


/* Topic: Attributes in Lists
	_____________________________________________

	--- Code
	[[AttributeA, AttributeB(), AttributeC(12, "string")]]
	[[Namespace::AttributeA, Namespace::AttributeB(), Namespace::AttributeC(12, "string")]]
	---
*/


/* Topic: Attributes with Using
	_____________________________________________

	--- Code
	[[using Namespace: AttributeA]]
	[[using Namespace: AttributeA, AttributeB(), AttributeC(12, "string")]]
	---
*/
